src/services/user.service.ts

```diff
@@ -0,0 +1,49 @@
+import { User } from "../models";
+import { UserRepository } from "../repositories";
+
+export class UserService {
+  constructor(private readonly userRepository: UserRepository) {}
+
+  getUsers(): User[] {
+    return this.userRepository.getUsers();
+  }
+
+  getUserByID(userId: string): User {
+    return this.userRepository.getUserByID(userId);
```

Comment:
I'm not sure that repository is a necessary layer

Reply:
Probably, Repository is a little excessive in our case. If you don't mind I would leave it like it is and exclude it in the next homework.